app/views/passengers/_check_existing.haml

```diff
@@ -0,0 +1,9 @@
+.modal-dialog{role: 'document'}
+  .modal-content
+    .modal-header
+      %h5 A passenger already exists for this spire ID. Go to edit that passenger?
+      %button.close{data: {dismiss: 'modal', 'aria-label':'Close'}}
+        %span{'aria-hidden': 'true'} &times;
+    .modal-footer
+      %button.btn.btn-default{data: {dismiss: 'modal', 'aria-label':'Close'}} Keep adding new
```

Comment:
What does `.btn-default` do?

Reply:
Why I chose it: It had the color I wanted and it seemed to fit lexically as well since it's a "dismissal" like, no, stay with the default action which is to just keep going. 
As for what it "does" it just provides styling, that's it.

Reply:
Yeah. My question came from the fact that I didn't know we were still on Bootstrap 3 on this app.  Bootstrap 4 doesn't have this class.